harvester_e2e_tests/apis/test_settings.py

```diff
@@ -162,3 +162,35 @@ def test_invalid_S3(self, api_client):
             f"S3 backup-target should check key/secret/bucket/region"
             f"API Status({code}): {data}"
         )
+
+
+@pytest.mark.p0
+@pytest.mark.negative
+@pytest.mark.settings
+@pytest.mark.settings
```

Comment:
Duplicated mark.

Reply:
Great catch :smile: :+1: - I've cleaned up the extra decorator - thanks for that mention